react-with-graphql-tutorial.md

```diff
@@ -1795,9 +1796,9 @@ const resolveAddStarMutation = mutationResult => state => {
 };
 {{< /highlight >}}
 
-Now try to star the repository again. You may have to go on the Github page for the repository first to unstar it. Nevertheless, the button label should adapt to the updated `viewerHasStarred` property from the local state. Thus the button should show either a "Star" or "Unstar" label.
+Now try to star the repository again. You may have to go on the Github page and unstar it first. The button label should adapt to the updated `viewerHasStarred` property from the local state to show a "Star" or "Unstar" label. You can use whate you've learned for starring repositories on implementing a `removeStar` mutation. 
 
-There is one little extra mile that you can take to improve the star mutation from a user experience perspective. The obvious part would be implementing the `removeStar` mutation. But that's your exercise at the end of this section. Another improvement could be showing the current count of people who have starred the repository and updating this count in the `addStar` (and `removeStar`) mutation. First, you can retrieve the total count of stargazers by adding the following fields to your query:
+ We also want to show the current number of people who have starred the repository and updated this count in the `addStar` (and `removeStar`) mutation. First, you can retrieve the total count of stargazers by adding the following fields to your query:
```

Comment:
Can we keep "There is one little extra mile that you can take to improve the star mutation from a user experience perspective. The obvious part would be implementing the `removeStar` mutation. But that's your exercise at the end of this section." in the beginning of the paragraph?

Reply:
I removed the part you asked me to put back it because it wasn't good.  "one little extra mile" is cliche' and doesn't mean anything, and that whole sentence tells the reader nothing about the current lesson.  The next part states the obvious by its own words "the obvious part", which again, adds nothing to the paragraph.  The last sentence starts with a "But"(incorrect) and tells the reader about an exercise coming up, which they'll figure out if they keep reading.  It also interrupts the flow of what you're teaching them now to tell them about something they'll learn later.  I think it's a digression that makes the whole paragraph harder to follow.

Reply:
Also, the sentence before it mentions adding the 'removeStar' mutation along with the 'addStar'.  I removed it from the parenthesis to make that sentence clearer.  If you're worried about the reader not knowing the 'removeStar' is coming up while covering the current 'addStar', it will make sense to them when they reach the exercises.